### PR TITLE
LPS-152541 BindSegmentsCompanyConfigurationMVCActionCommandTest

### PR DESCRIPTION
**Pull request by jkappler**
Original PR: https://github.com/liferay-tango/liferay-portal/pull/2566

### CODE REVIEW

**Comment by jkappler** [comment]
ci:test:sf

**Comment by jkappler** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
@cgoncas, @Stephy6, please let me know what you think please ❤️

**Comment by cgoncas** [comment]
ci:test:relevant

**Review by cgoncas — CHANGES_REQUESTED** [review]

**Comment by jkappler** [comment]
Que chulo @cgoncas, didn't know that we had this :D. Is there any way to set this on setup instead of each test?

**Comment by jkappler** [comment]
ci:test:relevant

File: modules/apps/segments/segments-test/src/testIntegration/java/com/liferay/segments/internal/portlet/action/test/BindSegmentsCompanyConfigurationMVCActionCommandTest.java [hunk 1]
```diff
@@ -62,124 +59,142 @@ public class BindSegmentsCompanyConfigurationMVCActionCommandTest {
...
-	public void setUp() throws Exception {
-		_configurationProvider.deleteCompanyConfiguration(
-			SegmentsCompanyConfiguration.class, _company.getCompanyId());
-	}
-
 	@Test
 	public void testProcessActionRoleSegmentationDisabled() throws Exception {
-		MockLiferayPortletActionRequest mockLiferayPortletActionRequest =
-			_getMockLiferayPortletActionRequest();
+		try (CompanyConfigurationTemporarySwapper
```
Comment (jkappler) [inline modules/apps/segments/segments-test/src/testIntegration/java/com/liferay/segments/internal/portlet/action/test/BindSegmentsCompanyConfigurationMVCActionCommandTest.java:64]: like this right @cgoncas?

**Comment by jkappler** [comment]
ci:reevaluate:1377401_3543

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
Sorry @cgoncas, renamed the branch and it was deleted, PR resent here: https://github.com/liferay-tango/liferay-portal/pull/2703

**Pull request closed without merging**